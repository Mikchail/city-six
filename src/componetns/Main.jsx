import React from "react";
import PropTypes from "prop-types";

const Main = ({data}) => {
  return (
    <div>
      Main
      {data.map((item) => {
        return <p key={item}>{item}</p>;
      })}
    </div>
  );
};

Main.propTypes = {
  data: PropTypes.array
};

export default Main;
