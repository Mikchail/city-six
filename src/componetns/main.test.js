import React from "react";
import renderer from "react-test-renderer";
import Main from "./Main";

const data = [
  `Beautiful & luxurious apartment at great location`,
  `Wood and stone place`
];

it(`<Main/> should render place`, () => {
  const tree = renderer.create(<Main data={data} />).toJSON();

  expect(tree).toMatchSnapshot();
});
