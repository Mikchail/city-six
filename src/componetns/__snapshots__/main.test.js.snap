// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render place 1`] = `
<div>
  Main
  <p>
    Beautiful & luxurious apartment at great location
  </p>
  <p>
    Wood and stone place
  </p>
</div>
`;
